js/apps/system/_admin/aardvark/APP/frontend/js/routers/router.js

```diff
@@ -801,8 +806,9 @@
     cInfo: function (colname) {
       const self = this;
 
+      window.SkeletonLoader.render()
       this.checkUser();
-
+      
```

Comment:
one small thing - maybe this change is unnecessary, just spaces

Reply:
Done! Thanks for catching it!